Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -29,5 +31,26 @@ public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetAllJokesAsync()
             return Ok(jokeDtos);
         }
 
+        [AllowAnonymous]
+        [HttpGet("{id}")]
+        public async Task<JokeReplyDto> GetJokeById(int id)
+        {
+            var jokeReplyDto = await _jokesService.GetJokeReplyById(id);
+            return jokeReplyDto;
+        }
+
+        // Create the whole line for Update Joke API call.
+        // UpdateJoke inside controller HTTP put
+        // UpdateJoke inside JokeService
+        // UpdateJoke inside JokeRepository
+        // Use AutoMapper if you can.
+
+        [HttpPatch]
+        [AllowAnonymous]
```

Comment:
This is a http put, patch is when your updating a single property and would require json patch. Should also be authorized